### PR TITLE
Specify host/container ports for InfluxDB, so that the UI is accessible

### PR DESCRIPTION
**Pull request by saad-ali**
CC @vishh

### CODE REVIEW

**Comment by vishh** [comment]
@saad-ali: Needs a rebase. Otherwise LGTM

**Comment by saad-ali** [comment]
PTAL rebased

File: deploy/kube-config/influxdb/v1beta3/influxdb-grafana-controller.json [hunk 1]
```diff
@@ -22,7 +22,15 @@
         "containers": [
           {
             "image": "kubernetes/heapster_influxdb:v0.3",
-            "name": "influxdb"
+            "name": "influxdb",
+            "ports": [
+              {
+                "containerPort": 8083
```
Comment (vishh) [inline deploy/kube-config/influxdb/v1beta3/influxdb-grafana-controller.json]: You have not specified any hostport. Why add just the container port?

Comment (saad-ali) [inline deploy/kube-config/influxdb/v1beta3/influxdb-grafana-controller.json]: Bad merge. Fixed.

**Comment by saad-ali** [comment]
PTAL.

**Comment by vishh** [comment]
LGTM. Thanks @saad-ali

**Pull request merged**